main.py

```diff
@@ -0,0 +1,44 @@
+import ssl
+import os
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
+from datetime import datetime, timedelta
+from urllib.parse import urlparse
+import telebot
+
```

Comment:
Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.

Also it's a good practise to group imports im alphabet order per such a block